.github/workflows/ic-ui-kit-develop.yml

```diff
@@ -5,6 +5,7 @@ on:
   push:
     branches:
       - develop
+      - v3.0.0/develop
```

Comment:
Do the storybook target folders need updating for v3.0.0/develop too? I think currently they'll both deploy to /develop/web-components etc.

Reply:
Fixed!